packages/components/src/props.tsx

```diff
@@ -1,61 +0,0 @@
-import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
-import { GetServerSidePropsContext, GetServerSidePropsResult } from 'next'
-import { APOLLO_STATE_PROP_NAME, getClient } from './client'
-import { COOKIE_ADDRESS, COOKIE_JWT_TOKEN } from './session'
-
-type UserProp = {
-  address: string | null
-  token: string | null
-}
-
-export type PropsWithUser<T> = T & {
-  user: UserProp
-}
-
-type GetServerSidePropsContextWithUser = GetServerSidePropsContext & {
-  user: UserProp
-}
-
-export function wrapServerSideProps<T extends { [key: string]: unknown }>(
```

Comment:
This was only used in templates so has moved there. This may be overkill now and might be deleted later on